.github/workflows/dockerhub.yml

```diff
@@ -2,8 +2,10 @@ name: Dockerhub Pipeline
 
 on:
   push:
+    branches:
+      - master
```

Comment:
that is actually redundant, because it generates tags only on default branch, and if there are no tags it won't push to dockerhub. But it is ok for clarity

Reply:
Though I thought that we removed `type=ref,event=pr` here https://github.com/aeternity/ae_mdw/pull/1095/files#diff-79be07d4634c4c8e630c58e010635d3621a19f63e92861570af1570120df38b6R30

Reply:
Even if there was a commit on master without tags? Since there is a master [here](https://hub.docker.com/r/aeternity/ae_mdw/tags) it might the case to also push under any condition or remove the master.

Reply:
it is supposed to push any commit to `master` as `master` and `v*` only when tagged